run_dir/design/element_flowcells.html

```diff
@@ -0,0 +1,122 @@
+
+<div id="element_fc_table_div">
+    <h1>
+        <span id="element_page_title">Element BioSciences (AVITI) flowcells</span>
+    </h1>
+
+    <table class="table table-striped table-bordered sortable" id="element_fc_table">
+
+        <thead style="vertical-align: top">
+            <tr class="sticky darkth">
+                <th class="sort" data-sort="start_date">Start date</th>
+                <th class="sort" data-sort="run_name">Run name</th>
+                <th class="sort" data-sort="run_type">Run type</th>
+                <th class="sort" data-sort="side">Side</th>
+                <th class="sort" data-sort="cycles">Cycles</th>
+                <th class="sort" data-sort="throughput_selection">Throughput Selection</th>
+                <th class="sort" data-sort="kit_configuration">Kit Configuration</th>
+                <th class="sort" data-sort="chemistry_version">Chemistry Version</th>
+                <th class="sort text-center" data-sort="outcome">Outcome</th>
+            </tr>
+        </thead>
+        <tfoot>
+            <tr class="darkth">
+                <th class="sort searchable" data-sort="start_date">Start date</th>
+                <th class="sort searchable" data-sort="run_name">Run name</th>
+                <th class="sort searchable" data-sort="run_type">Run type</th>
+                <th class="sort searchable" data-sort="side">Side</th>
+                <th class="sort searchable" data-sort="cycles">Cycles</th>
+                <th class="sort searchable" data-sort="throughput_selection">Throughput Selection</th>
+                <th class="sort searchable" data-sort="kit_configuration">Kit Configuration</th>
+                <th class="sort searchable" data-sort="chemistry_version">Chemistry Version</th>
+                <th class="sort text-center" data-sort="outcome">Outcome</th>
+            </tr>
+        </tfoot>
+        <tbody class="list">
+            {% for onefc in element_fcs %}
+                <tr>
+                    <td class="start_date">{{ onefc.key }}</td>
+                    <td class = run_name>
+                        <a class="text-decoration-none" href="/flowcells_element/{{ onefc.id }}">{{ onefc.value['RunName'] }}</a>
+                        {% if onefc.value.get('outcome') == 'OutcomeCompleted' %}
+                            <abbr title="Run completed"><span class="fa fa-check-circle"></span></abbr>
```

Comment:
I don't think the fa-symbols are showing up?

Reply:
Good catch! It was a typo and should be fixed now.